ansible/azure-aap-deployment-driver
# Add Region, AccessType and Deployment End Time metrics to Telemetry

# Description

This PR is for adding the following metrics to the telemetry payload:
1. Region/Location of deployment.
2. AccessType (Public/Private)
3. Deployment End Time (RTC3339 / ISO 8601 format)



## PR task checklist

- ~[] I have added thorough tests~


# Ticket

https://issues.redhat.com/browse/AAP-10508
https://issues.redhat.com/browse/AAP-10507
https://issues.redhat.com/browse/AAP-10482


# Testing

1. Pull down the PR.
2. Create a new deployment using the installerincluded branch of aap-azurerm repo.
3. Once the Deployment is complete, go to Segment (https://app.segment.com/redhat-devtools/sources/ansible_on_azure_managed_app_dev/debugger) and check if the event is published here.
4. Check for the properties published in the PR.
